unlock-app/src/components/interface/buttons/layout/About.js

```diff
@@ -3,7 +3,7 @@ import Svg from '../../svg'
 import { LayoutButton } from '../Button'
 
 const About = props => (
-  <LayoutButton href="/about" title="About" {...props}>
+  <LayoutButton href="/about" label="About" {...props}>
```

Comment:
Makes sense to use label here on the layout buttons - but let's continue to use `title` for UI elements?

Reply:
ok!